packages/testsweets/lib/src/models/application_models.dart

```diff
@@ -15,37 +15,38 @@ class WidgetDescription with _$WidgetDescription {
     String? id,
 
     /// The name of the view this widget was captured on
-    String? viewName,
+    required String viewName,
 
     /// The orignal name of the view this widget was captured on before the prettify
-    String? originalViewName,
+    required String originalViewName,
 
     /// The name we want to use when referring to the widget in the scripts
     @Default('') String name,
 
     /// The type of the widget that's being added
-    WidgetType? widgetType,
+    required WidgetType widgetType,
 
     /// The position we defined for he widget
-    WidgetPosition? position,
+    required WidgetPosition position,
```

Comment:
Thanks, every widget requires a position. Even the view, it's position is 0,0 so that it's out of the way but still on screen. Remember we don't actually know what view we're on, it's just an abstraction we placed ontop of widget description to make it easier for us to track. But it's just a widget with a position 0. It's important that our models reflect what we expect from its usage in the code.

Reply:
You're right that's much better, I decided to make the widget position nullable first but later I found a way to drop that but I forget to do it